json_serializable/lib/src/shared_checkers.dart

```diff
@@ -62,6 +62,9 @@ String asStatement(DartType type) {
   return ' as $typeCode';
 }
 
+// ignore: deprecated_member_use
```

Comment:
Is there an issue filed for the migration?

Reply:
Wasn't planning on it. When we want to roll forward our analyzer dependency (and these members go away) we'll have to fix 'em